src/app/workspace/resource/resource.component.ts

```diff
@@ -243,56 +249,78 @@ export class ResourceComponent implements OnInit, OnChanges, OnDestroy {
 
-                } else {
-                    // if there is no incomingResource and the resource has a still image property, assign the iiiUrl to be passed as an input to the still-image component
-                    if (!this.incomingResource && this.resource.res.properties[Constants.HasStillImageFileValue]) {
-                        this.iiifUrl = (this.resource.res.properties[Constants.HasStillImageFileValue][0] as ReadStillImageFileValue).fileUrl;
-                    }
+                    // get the parent resource to display
+                    this.getResource(annotationOfIri);
 
-                    this.selectedTabLabel = this.resource.res.entityInfo?.classes[this.resource.res.type].label;
+                    // open annotation`s tab and highlight region
+                    this.selectedTabLabel = 'annotations';
+                    this.openRegion(iri);
 
-                    // get information about the logged-in user, if one is logged-in
-                    if (this._session.getSession()) {
-                        this.session = this._session.getSession();
-                        // is the logged-in user project member?
-                        // --> TODO: as soon as we know how to handle the permissions, set this value the correct way
-                        this.editPermissions = true;
-                        // is the logged-in user system admin or project admin?
-                        this.adminPermissions = this.session.user.sysAdmin ? this.session.user.sysAdmin : this.session.user.projectAdmin.some(e => e === res.res.attachedToProject);
-                    }
+                    this.selectedRegion = iri;
+                    // define resource as annotation of type region
+                    this.resourceIsAnnotation = (this.resource.res.entityInfo.classes[Constants.Region] ? 'region' : 'sequence');
 
-                    this.representationsToDisplay = this.collectRepresentationsAndAnnotations(this.resource);
+                } else {
 
-                    if (!this.representationsToDisplay.length && !this.compoundPosition) {
-                        // the resource could be a compound object
-                        this._incomingService.getStillImageRepresentationsForCompoundResource(this.resource.res.id, 0, true).subscribe(
-                            (countQuery: CountQueryResponse) => {
+                    if (response.isDeleted) {
+                        // deleted resource; no further infos needed
 
-                                if (countQuery.numberOfResults > 0) {
-                                    this.compoundPosition = new DspCompoundPosition(countQuery.numberOfResults);
-                                    this.compoundNavigation(1);
-                                } else {
-                                    // not a compound object
+                    } else {
```

Comment:
Will there be more line within this if-body? If not you can negate it and transfer the content of else-body

Reply:
I will add a todo-comment. Because this is still an open point. How do we display a deleted/archived resource?

Reply:
I honestly don't know